apps/settings/elements/wifi_enter_certificate_nickname.html

```diff
@@ -10,13 +10,11 @@ <h1 data-l10n-id="enterCertificateNickname"> Enter certificate nick name </h1>
     <div>
       <ul>
         <li class="nickname">
-          <p data-l10n-id="nickname">Nick name</p>
-          <input class="certificate-file-nickname" type="text" data-ignore name="nickname" dir="auto" />
+          <p data-l10n-id="nickname"></p>
+          <input class="certificate-file-nickname" type="text" dir="auto" />
         </li>
         <li class="description">
-          <p data-l10n-id="enter-certificate-name-msg">
-            You can change the nick name or keep the default name.
-          </p>
+          <p data-l10n-id="enter-certificate-name-msg"></p>
         </li>
```

Comment:
please help to remove all innerHTML from elements which has `data-l10n-id` in this html, thanks !